src/main/java/com/amihaiemil/zold/PayloadOf.java

```diff
@@ -39,7 +39,6 @@
  * @author George Aristy (george.aristy@gmail.com)
  * @version $Id$
  * @since 0.0.1
- * @todo #25:30min Add tests for PayloadOf.
```

Comment:
@denis-konovalyenko why this todo was removed?

Reply:
@g4s8 , this is the scope of the pull request #53.

Reply:
@denis-konovalyenko I understand it,  but why to include these changes in this PR? It's not good practice, because when you change one PR you also should update another, it's more difficult to review PR, because ther's more changes than required.

Reply:
@g4s8 , I totally agree with you. We had had difficulties in merging the approved pull request #53 for some time. So, I decided to move on with issue #34.